forms-flow-web/src/components/Form/List.scss

```diff
@@ -12,17 +12,18 @@
 }
 
 .published-forms-label{
-    color: #52C41A;
-    background: #EAFBE7 0% 0% no-repeat padding-box;
-    border: 1px solid #52C41A;
+    color: var(--published-form-label);
+    // #52C41A;
+    background: var(--published-background) 0% 0% no-repeat padding-box;
+    border: 1px solid var(--published-form-label);
     border-radius: 7px;
 }
 
 .unpublished-forms-label{
-    background: #E6F7FF 0% 0% no-repeat padding-box;
+    background: var(--unpublished-background) 0% 0% no-repeat padding-box;
```

Comment:
Here we can use a specific color like primary or color-primary-light some thing like that no need to bring specific name it will be very difficult to manage.